### ISSUE TITLE
Using of logged accounts

### ISSUE THREAD

**Issue by logan478** [AssetKid/raider-release#51]
Can you please somehow make a way that we can choose which logged account will Raider use? Because now if you have logged more accounts, and one of them have rate limit, then Raider is not using even those without rate limit. So i must delete accounts with rate limit and then log in again after day. And this is bad because Instagram will block your account if you doing it often and they sending you security mails "uknown device" logged in.

**Comment by ayynonwhymouse** [comment]
I just came here to post about this after getting the rate-limited message even though I'm trying to download from an IG that is followed by my other non-rate limited account. It would be amazing to be able to tell Raider which account to use.

**Comment by ayynonwhymouse** [comment]
@logan478 So I have a workaround for this for the time being. Have each of your accounts follow a separate private account. When you want to switch the default account you're using, first update the private account corresponding to the account you wish to use. For example,

My accounts are "A" & "B", "C" and "D" are private accounts, "E" is either a public account or an private account followed by both A & B.

Have A follow C, and B follow D.

If I want to update E from account A, first I update C.
If I want to update E from account B, first I update D.

The account you're using will remain the default until you try to update an account that it doesn't have access to.

**Comment by logan478** [comment]
I think something has changed, because now i was able to update 750 subs with one account, without rate limit message. If it will be now like that forever then changing accounts doesnt matter anymore. My settings were turn off highlights and 5 seconds delay. Before i had always rate limit message at 500 subs.

**Comment by supersick123** [comment]
> I think something has changed, because now i was able to update 750 subs with one account, without rate limit message. If it will be now like that forever then changing accounts doesnt matter anymore. My settings were turn off highlights and 5 seconds delay. Before i had always rate limit message at 500 subs.

Yeah, I have the same settings as you. I used to have a 60kb acc file with 600+ subs, and 1 account can download them all when that was previously not possible. However, I think downloading that many accounts in a short period of time will increase the likelihood of being marked for suspicious activity, given the recent changes in the past 2 weeks.

**Comment by supersick123** [comment]
> So i must delete accounts with rate limit and then log in again after day. And this is bad because Instagram will block your account if you doing it often and they sending you security mails "uknown device" logged in.

If I can make a suggestion/addition to this enhancement... if @AssetKid is willing, would it be possible for Raider to have a "toggle" on/off switch for accounts? It might be easier for us to login under multiple accounts all at once and then just disable all the accounts we don't want to use at the moment (leaving only one of them to be used). 

This will reduce the need to login multiple times using the same account (ie once a day), which results in less "unknown device" login messages/flags for suspicious activity (thereby reducing the likelihood of the problem mentioned in #67).
